pom.xml

```diff
@@ -665,8 +665,7 @@
                 <quarkus.package.type>native</quarkus.package.type>
                 <quarkus.native.container-build>true</quarkus.native.container-build>
                 <quarkus.native.native-image-xmx>4g</quarkus.native.native-image-xmx>
-                <!-- please keep in mind that lifecycle-application module also defines quarkus.native.builder-image property -->
```

Comment:
This line was dropped because 677520f7de0ac254a2d7d51bf75ee58acb2fb4a4 removed the property from lifecycle app